### DISCUSSION TITLE
GitHub action hangs at the beginning of a step, can't see how to diagnose the problem

### DISCUSSION THREAD

**Discussion by vsajip** [community/community#5675] [Actions]
[This workflow](https://github.com/vsajip/cfgclient/runs/3621197038?check_suite_focus=true) completes without issue on Linux and OSX, but hangs on Windows without showing any progress at the beginning of a step:
![ss11](https://user-images.githubusercontent.com/130553/133622064-355460be-82bb-42f8-adaf-5fe6a663c6ba.png)
I've enabled additional debugging which shows up on earlier steps, but on the hanging step I don't even get the `##[debug]Evaluating condition for step` which is the first debug message shown in all of the earlier steps. Don't see how it could be part of the step itself, as that doesn't even seem to get going.

**Comment by bloomsirenix** [comment]
I recommend contacting support

**Reply by bloomsirenix** [reply]
sorry just realised that is a support feature, sorry

**Comment by vsajip** [comment]
I've [logged a support issue](https://support.github.com/ticket/1313438). The step eventually completed - took well under 5 mins under Ubuntu and OSX, but over _3 hours_ :astonished: on Windows! The delay happens _before_ the invoked code of the actual step starts running. Repeatable across at least two workflow runs.
